CONTRIBUTING.md

```
@@ -0,0 +1,57 @@
+
+### Code that should not be changed
+Our code is in sync with the [Stripe React Native SDK](https://github.com/stripe/stripe-react-native) we only maintain the Flutter bindings and the native code is a close copy of the Stripe native SDKs to ensure security and stability. Because of this we do not accept changes in the Stripe native SDK. The native sdk code is located in the `packages/stripe_android/android/src/main/kotlin/com/reactnativestripesdk` and `packages/stripe_ios/ios/stripe_ios/Sources/stripe_ios/Stripe Sdk` folders.
+
+### Useful commands
+
+- Format `melos run format`
+- Analyze `melos run analyze`
+- Test `melos run unittest`
+- Pub get `melos run get`
+
+##### Publishing
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h4; Actual: h5

(MD001, heading-increment)